Update some API changes:

fabriziomello opened the pull request:
- `add_continuous_aggregate_policy`: added new optional argument `include_tiered_data` introduced by 2.18.0.
- `refresh_continuous_aggregate`: added new optional argument `force` introduced by 2.18.0

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/timescale/docs?pullRequest=3759) <br/>All committers have signed the CLA.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/timescale/docs?pullRequest=3759) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/timescale/docs?pullRequest=3759) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: fabriziomello<br/>:white_check_mark: jamessewell<br/>:x: Ryan Booz<br/><hr/>**Ryan Booz** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/timescale/docs?pullRequest=3759) it.</sub>

fabriziomello committed:
Update some API changes:

- `add_continuous_aggregate_policy`: added new optional argument
  `include_tiered_data` introduced by 2.18.0.
- `refresh_continuous_aggregate`: added new optional argument `force`
  introduced by 2.18.0.

Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>

gayyappan commented on api/add_continuous_aggregate_policy.md:
```diff
@@ -49,6 +48,8 @@ about how continuous aggregates use real-time aggregation, see the
 |-|-|-|
 |`if_not_exists`|BOOLEAN|Set to `true` to issue a notice instead of an error if the job already exists. Defaults to false.|
+|`timezone`|TEXT|A valid time zone. If `initial_start` is also specified, subsequent executions of the refresh policy will be aligned on its initial start. However, daylight savings time (DST) changes may shift this alignment. Set to a valid time zone if this is an issue you want to mitigate. If omitted, UTC bucketing is performed. Defaults to `NULL`.|
+| `include_tiered_data` | BOOLEAN | Set true to read tiered data even if `timescaledb.enable_tiered_reads` is false. Defaults to `TRUE`. |
```
I thought the default is to use the current timescaledb.enable_tiered_data settings. @zilder ? 
```suggestion
| `include_tiered_data` | BOOLEAN | Enable/disable reading tiered data. This setting helps override the current settings for the`timescaledb.enable_tiered_reads` GUC. The default is NULL i.e we use the current setting for `timescaledb.enable_tiered_reads` GUC  | |
```

zilder replied:
Yes, it defaults to NULL meaning the behavior is defined by the `timescaledb.enable_tiered_reads` configuration.

mkindahl commented on api/refresh_continuous_aggregate.md:
```diff
@@ -62,6 +62,12 @@ not take place when buckets are materialized with no data changes or with
+
+|Name|Type|Description|
+|-|-|-|
+| `force` | BOOLEAN | Force the refresh even if the window range is already refreshed. Default to `FALSE`.|
```
Might be good to emphasize that it is an expensive operation, e.g., something like this:
```suggestion
| `force` | BOOLEAN | Force refresh of every bucket in the time range even if the bucket is already refreshed. Can be very expensive if a lot of data is refreshed. Default to `FALSE`.|
```

mkindahl commented on api/refresh_continuous_aggregate.md:
````diff
@@ -93,6 +99,12 @@ END
+`2020-02-01` exclusive even if it is already refreshed.
+
+```sql
+CALL refresh_continuous_aggregate('conditions', '2020-01-01', '2020-02-01', TRUE);
````
Probably good to encourage named parameters to avoid problems in the future.
```suggestion
CALL refresh_continuous_aggregate('conditions', '2020-01-01', '2020-02-01', force => TRUE);
```

fabriziomello committed:
Update api/add_continuous_aggregate_policy.md

Co-authored-by: gayyappan <45599764+gayyappan@users.noreply.github.com>
Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>

fabriziomello committed:
Update api/refresh_continuous_aggregate.md

Co-authored-by: Mats Kindahl <mats@timescale.com>
Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>

fabriziomello committed:
Update api/refresh_continuous_aggregate.md

Co-authored-by: Mats Kindahl <mats@timescale.com>
Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>

billy-the-fish commented on api/refresh_continuous_aggregate.md:
```diff
@@ -62,6 +62,12 @@ not take place when buckets are materialized with no data changes or with
+
+|Name|Type|Description|
+|-|-|-|
+| `force` | BOOLEAN | Force refresh of every bucket in the time range even if the bucket is already refreshed. Can be very expensive if a lot of data is refreshed. Default to `FALSE`.|
```
```suggestion
| `force` | BOOLEAN | Force refresh every bucket in the time range between `window_start` and `window_end`, even when the bucket has already been refreshed. This can be very expensive when a lot of data is refreshed. Default is `FALSE`.|
```

billy-the-fish commented on api/refresh_continuous_aggregate.md:
````diff
@@ -93,6 +99,12 @@ END
 $$;
 ```
 
+Forcing the refresh of the continuous aggregate `conditions` between `2020-01-01` and
````
```suggestion
Force the  `conditions` continuous aggregate to refresh between `2020-01-01` and
```

billy-the-fish commented on api/refresh_continuous_aggregate.md:
````diff
@@ -93,6 +99,12 @@ END
 ```
 
+Forcing the refresh of the continuous aggregate `conditions` between `2020-01-01` and
+`2020-02-01` exclusive even if it is already refreshed.
````
```suggestion
`2020-02-01` exclusive, even if the data has already been refreshed.
```

billy-the-fish commented on api/add_continuous_aggregate_policy.md:
```diff
@@ -49,6 +48,8 @@ about how continuous aggregates use real-time aggregation, see the
 |Name|Type|Description|
 |-|-|-|
 |`if_not_exists`|BOOLEAN|Set to `true` to issue a notice instead of an error if the job already exists. Defaults to false.|
+|`timezone`|TEXT|A valid time zone. If `initial_start` is also specified, subsequent executions of the refresh policy will be aligned on its initial start. However, daylight savings time (DST) changes may shift this alignment. Set to a valid time zone if this is an issue you want to mitigate. If omitted, UTC bucketing is performed. Defaults to `NULL`.|
```
```suggestion
|`timezone`|TEXT|A valid time zone. If you specify `initial_start`, subsequent executions of the refresh policy are aligned on `initial_start`. However, daylight savings time (DST) changes may shift this alignment. If this is an issue you want to mitigate, set `timezone` to a valid time zone. Default is `NULL`, [UTC bucketing](https://docs.timescale.com/use-timescale/latest/time-buckets/about-time-buckets/) is performed.|
```

billy-the-fish reviewed (approved):
Couple of small language things. Your call.

fabriziomello committed:
Update api/refresh_continuous_aggregate.md

Co-authored-by: Iain Cox <iain@timescale.com>
Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>

fabriziomello committed:
Update api/add_continuous_aggregate_policy.md

Co-authored-by: Iain Cox <iain@timescale.com>
Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>

fabriziomello committed:
Update api/refresh_continuous_aggregate.md

Co-authored-by: Iain Cox <iain@timescale.com>
Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>

fabriziomello committed:
Update api/refresh_continuous_aggregate.md

Co-authored-by: Iain Cox <iain@timescale.com>
Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>
